### ISSUE TITLE
[Bug] Kibana `ca_certs` connection option not working

### ISSUE THREAD

**Issue by unclestacks** [elastic/terraform-provider-elasticstack#582]
**Describe the bug**
With the latest release, we tried to use the `ca_certs` connection parameter with kibana, but we receive tls connection errors

**To Reproduce**
1. TF configuration used 
```
provider "elasticstack" {
  kibana {
    endpoints = ["https://kibana-stage.company.com"]
    username  = var.admin_user
    password  = var.admin_creds
    ca_certs  = ["/etc/ssl/trust/IssuingCA.pem"]
  }
}
```
2. TF operations to execute to get the error: `terraform apply`
3. See the error in the output:
```
│ Error: Failed to create data view
│
│   with module.elasticstack.elasticstack_kibana_data_view.data_views["default-filebeat"],
│   on ../../../modules/elasticstack/kibana_data_view.tf line 33, in resource "elasticstack_kibana_data_view" "data_views":
│   33: resource "elasticstack_kibana_data_view" "data_views" {
│
│ Post "https://kibana-stage.company.com/s/default/api/data_views/data_view": tls: failed to verify certificate: x509: certificate signed by unknown authority
```

**Expected behavior**
provider should be able to establish a secure connection to Kibana

**Versions (please complete the following information):**
 - OS: Linux
 - Terraform Version: v1.3.7
 - Provider version: v0.11.1
 - Elasticsearch Version: v8.12.1

**Additional context**
I don't believe kibana `ca_certs` is present in the release despite being in the release notes

**Comment by AdilKhan1117** [comment]
Is there, any update for this, also seeing the same issue? Thanks, would be great to have a more stable version for this provider!

**Comment by tobio** [comment]
@AdilKhan1117 are you able to confirm which resources you're seeing this error on?

**Comment by AdilKhan1117** [comment]
> @AdilKhan1117 are you able to confirm which resources you're seeing this error on? 

Thanks 

It's for the below, 


Nested Schema for Kibana:

ca_certs (List of String) A list of paths to CA certificates to validate the certificate presented by the Kibana server.

insecure (Boolean) Disable TLS certificate validation

**Comment by unclestacks** [comment]
We've also seen this issue trying to manage `elasticstack_kibana_slo` resources.

**Comment by tobio** [comment]
@unclestacks @AdilKhan1117 this should be fixed in 0.11.2
